nrf_802154/driver/src/nrf_802154_trx.c

```diff
@@ -426,6 +457,12 @@ void nrf_802154_trx_enable(void)
 
     nrf_radio_reset();
 
+    // Apply DEVICE-CONFIG-254 if needed.
```

Comment:
```suggestion
    /* Apply DEVICE-CONFIG-254 if needed. */
```

Reply:
This comment style is consistent with the one used inside the function. Please see next 50 lines or so.

Reply:
I don't see it, we always use this type of commenting in NCS.

Reply:
@czeslawmakarski, could you weight in on this one as a code owner? Whether we shall stick to the style currently used in the Radio Driver, or maybe comment style shall be updated in the file?

Reply:
Folder `nrf_802154` sticks to the Nordic C Firmware Coding Standard, not the NCS Coding Standard. Nordic C Firmware Coding Standard does allow single line C++-style comments (as an alternative to classic C-style comments). Though personally I am not a big fan of C++-style comments, I think we should port them from C++ to C style in a batched manner. Let's leave it as is for now, and sometime later we can provide the PR with batch porting of comments.